catalog/FriendScreen.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useMemo } from 'react'
+import {
+  StyleSheet,
+  Text,
+  View,
+  SafeAreaView,
+  Alert,
+  ScrollView,
+} from 'react-native'
+import useTheme from '../components/hooks/useTheme'
+import Friend from '../components/molecules/Friend/Friend'
+import GameCard from '../components/molecules/GameCard/GameCard'
+
+const commonFriendData = {
+  imageUrl: 'https://i.imgur.com/LK7ymaN.png',
+  name: 'Lily Woofers 10',
```

Comment:
@J-Rehan It's a small thing, but names will only ever have one space so it's more realistic for this to be "Lily Woofers10". All usernames are `[firstname] [nickname]`